src/transport/SessionManager.h

```diff
@@ -197,6 +197,12 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
      */
     void Shutdown();
 
+    /**
+     * @brief
+     * Sync the groupCounter when a fabric is removed from the fabric table
```

Comment:
This should document that this is a notification that the fabric was removed.  What session manager does with that information is implementation details, no?